config/manifests/prod/bases/logic-operator-rhel8.clusterserviceversion.yaml

```diff
@@ -289,13 +289,13 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-  - image: registry.redhat.io/openshift-serverless-1/logic-operator-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-operator-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_OPERATOR
-  - image: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-swf-devmode-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_DEVMODE
-  - image: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.34.0
+  - image: registry.redhat.io/openshift-serverless-1/logic-swf-builder-rhel8:1.35.0
     name: IMAGE_LOGIC_SWF_BUILDER
   - image: registry.redhat.io/openshift4/ose-kube-rbac-proxy@sha256:4564ca3dc5bac80d6faddaf94c817fbbc270698a9399d8a21ee1005d85ceda56
     name: IMAGE_KUBE_RBAC_PROXY
-  replaces: logic-operator-rhel8.v1.33.0
-  version: 0.0.0
+  replaces: logic-operator-rhel8.v1.34.0
+  version: logic-operator-rhel8.v1.35.0
```

Comment:
following the rational in: bundle.prod/manifests/logic-operator-rhel8.clusterserviceversion.yaml

shouldn't this version number here be, version: 1.35.0 instead of version: logic-operator-rhel8.v1.35.0 ?

@jakubschwan  @ricardozanini ?

Reply:
@wmedvede you're right. The version is `1.35`. I just checked here https://github.com/kiegroup/kogito-serverless-operator/blob/9.101.x-prod/bundle.prod/manifests/logic-operator-rhel8.clusterserviceversion.yaml#L897

@rgdoliveira The fix I made in the CSV is from this branch, correct?

Reply:
@ricardozanini the fix you did for CSV was done in branch 9.101.x-prod and I just brought it to main branch.

The file @wmedvede mentioned `bundle.prod/manifests/logic-operator-rhel8.clusterserviceversion.yaml` looks correct in 9.101.x-prod branch, see https://github.com/kiegroup/kogito-serverless-operator/blob/9.101.x-prod/bundle.prod/manifests/logic-operator-rhel8.clusterserviceversion.yaml#L897

But the other file `config/manifests/prod/bases/logic-operator-rhel8.clusterserviceversion.yaml` in fact had version as `0.0.0` for the bundle respin we did for OSL 1.35, https://github.com/kiegroup/kogito-serverless-operator/blob/9.101.x-prod/config/manifests/prod/bases/logic-operator-rhel8.clusterserviceversion.yaml#L287

As there was no issues during tests and it worked it the end, I suppose this second file was not really used in productization?

Reply:
I've just changed it to 1.35.0